softwerkskammer/test/members/beansLeak_test.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const expect = require('must-dist');
+const CoolBeans = require('CoolBeans');
+
+describe('configureForTest issue', () => {
+  it('allows to configure different beans per test', () => {
+      const moduleName = 'membersPersistence';
+      const module1a = require('../../testutil/configureForTest').get('beans').get(moduleName);
+      const module2 = require('../../testutil/configureForTestWithDB').get('beans').get(moduleName);
+      const module1b = require('../../testutil/configureForTest').get('beans').get(moduleName);
+      expect(module1a).to.not.be(module2);
+      expect(module1b).to.not.be(module2);
```

Comment:
Und ja, ich möchte gern module1a === module1b

Ansonsten konfigurieren wir in jedem Test wieder von vorne, das kann's auch nicht sein...